features/login/impl/src/main/kotlin/io/element/android/features/login/impl/root/LoginRootView.kt

```diff
@@ -348,3 +366,26 @@ private fun ContentToPreview(state: LoginRootState) {
         onBackPressed = {}
     )
 }
+
+@OptIn(ExperimentalComposeUiApi::class)
```

Comment:
Could you move this to `io.element.android.libraries.designsystem.theme.components.TextField.kt` in the `:libraries:designsystem` module?

Reply:
Did the move in 61a1d219a2a63d538eeb62d8540fe424eec8eac2 :)